checkout-core/src/main/java/com/adyen/checkout/core/internal/data/model/JsonUtils.kt

```diff
@@ -95,4 +132,41 @@ object JsonUtils {
             }
         }
     }
+
+    /**
+     * Parses a [JSONArray] to a list of integers.
+     *
+     * @param jsonArray The JSONArray to be read.
+     * @return A [List] of integers, or null if the jsonArray was null.
+     */
+    @JvmStatic
+    fun parseOptIntegerList(jsonArray: JSONArray?): List<Int>? {
+        if (jsonArray == null) {
+            return null
+        }
+        val list: MutableList<Int> = ArrayList()
+        for (i in 0 until jsonArray.length()) {
+            val item = jsonArray.optInt(i, -1)
```

Comment:
Why are we using -1 here? If a value in this array is not an integer we should throw an exception, -1 could hide an integration error we have